File: web3rpc/rpc-specs/code-samples/javascript/admin/setSpamThrottlerWhiteList.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const address = ['0xfdeedbb2fe5b48d5b49e435ba00e0358740d0cf5']
    provider.admin.setSpamThrottlerWhiteList(address, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/admin/setSpamThrottlerWhiteList.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = ["0xfdeedbb2fe5b48d5b49e435ba00e0358740d0cf5"]

w3 = Web3(Web3.HTTPProvider(host))
admin_response = w3.geth.admin.set_spam_throttler_white_list(address)

print(admin_response)
